### ISSUE TITLE
Question about the test results..

### ISSUE THREAD

**Issue by jdg900** [zhwhong/UIDNet#2]
I divided the CBSD500 data set into training set 432 and test set 68. I generated 54,432 64x64 patches from the training set. I trained the bsd_model.py according to natural images/bsd_train.sh.
However, when I test the model using 321x481 or 481x321 whole images of CBSD68, the PSNR and SSIM are not similar to the results you reported. I want to know if I am missing anything. Did you test the model using 64x64 patches?

Thank you.
